mvp_project/README.md

```diff
@@ -1,109 +1,183 @@
 
-## Категория 3: Интерактивные функции
+2. **Авторизация зарегистрированного пользователя**  
+   _Как пользователь, я хочу войти в свой аккаунт, чтобы работать с дневниками._  
+   - **Контекст**: Ввод email и пароля, получение доступа через JWT-токен.  
+   - **Технически**: FastAPI проверяет данные и выдаёт токен.  
 
-### 7. Визуализация маршрутов
-- **Как пользователь**, я хочу видеть карту с отмеченными местами, чтобы наглядно отслеживать свои путешествия.
-  - **Контекст:** Пользователь открывает дневник, и карта отображает посещённые точки с медиафайлами.
-  - **Технически:** Используется Google Maps API или Mapbox для рендеринга карт.
+3. **Авторизация через социальные сети**  
+   _Как пользователь, я хочу входить через Google/Facebook для удобства._  
+   - **Контекст**: OAuth2 для авторизации, сохранение данных в PostgreSQL.  
 
-### 8. Создание видеоисторий
-- **Как пользователь**, я хочу создавать видеоистории из своих записей, чтобы делиться ими в социальных сетях.
-  - **Контекст:** Пользователь выбирает дневник, шаблон, добавляет музыку и экспортирует видео.
-  - **Технически:** Генерация видео происходит на сервере с использованием библиотек, таких как FFmpeg.
+4. **Сброс пароля**  
+   _Как пользователь, я хочу сбросить пароль, чтобы восстановить доступ._  
+   - **Контекст**: Получение ссылки для сброса пароля через email.  
+   - **Технически**: Генерация временного токена, обновление пароля в базе.  
 
 ---
 
-## Категория 4: AI-функции
+### **Категория 1: Управление дневником путешествий**
+1. **Создание дневника**  
+   _Как пользователь, я хочу создать новый дневник, чтобы структурировать свои путешествия._  
+   - **Контекст**: Добавление названия, описания, обложки.  
+   - **Технически**: FastAPI сохраняет данные дневника в PostgreSQL.  
+
+2. **Добавление мультимедиа**  
+   _Как пользователь, я хочу загружать фото и видео для визуализации записей._  
+   - **Контекст**: Загрузка файлов, их просмотр перед сохранением.  
+   - **Технически**: FastAPI отправляет файлы в облачное хранилище (AWS S3).  
+
+3. **Организация записей**  
+   _Как пользователь, я хочу упорядочивать записи по дням и местам._  
+   - **Контекст**: Добавление даты и местоположения к записи.  
+   - **Технически**: Записи связываются с дневником через внешние ключи.  
 
-### 9. Автоматическое описание мест
-- **Как пользователь**, я хочу, чтобы приложение распознавало достопримечательности на моих фотографиях и добавляло их описание, чтобы экономить время.
-  - **Контекст:** При загрузке фотографии приложение предлагает автоматическое описание.
-  - **Технически:** Используются API для анализа изображений, например Google Vision API.
+4. **Автоматическое отслеживание маршрутов**  
+   _Как пользователь, я хочу автоматически записывать маршруты с использованием GPS._  
+   - **Контекст**: Включение GPS для реального времени.  
+   - **Технически**: FastAPI принимает данные через WebSocket или POST-запросы.  

```

Comment:
That might be out of scope for MVP. Here would need to figure out how to access GPS tracking on devices.